assets/charts/components/headlamp/templates/NOTES.txt

```diff
@@ -0,0 +1,21 @@
+1. Get the application URL by running these commands:
```

Comment:
Oh wow, this is hard to read :smile:

Reply:
BTW, this won't be visible anyway to the user anyway.

Reply:
It was automatically generated by Helm, I can remove it if you want.

Reply:
I usually remove all files from `templates` when creating new chart to keep things simple, if the chart is not intended for external use. This allows keeping things very simple.